pixi.toml

```diff
@@ -0,0 +1,104 @@
+[workspace]
+channels = ["conda-forge", "nodefaults"]
+name = "modflow6-examples"
+platforms = ["win-64", "linux-64", "linux-aarch64", "osx-arm64", "osx-64"]
+version = "1.0.0"
+description = "This repository contains MODFLOW 6 examples."
+
+[tasks]
+# General
+script = { cmd = "pytest -v -n auto test_scripts.py -k", cwd = "autotest"} # Add script name e.g. pixi run script ex-gwe-ates.py
```

Comment:
again my preference would be to hardcode as few arguments to pytest as possible. what if the user just wants to run all the scripts? right now they'd get

```
$ pixi run script
ERROR: usage: pytest [options] [file_or_dir] [file_or_dir] [...]
pytest: error: argument -k: expected one argument
```

And if these are to be singular, it doesn't make sense to have `-n auto`.

Suggest keeping them plural and removing `-k`

- `scripts = "pytest -v -n auto test_scripts.py"`
- `notebooks = "pytest -v -n auto test_notebooks.py"`

Could even do it without `-n auto` but that can be opted out of with `-n 0` to follow the pattern in mf6.